ddtrace/internal/tracemethods.py

```diff
@@ -1,19 +1,68 @@
-import typing  # noqa:F401
+from typing import List
+from typing import Tuple
 
 import wrapt
 
+from ddtrace.internal.utils.deprecations import DDTraceDeprecationWarning
+from ddtrace.vendor.debtcollector import deprecate
 
-def _parse_trace_methods(raw_dd_trace_methods):
-    # type: (str) -> typing.List[str]
-    """Return the methods to trace based on the specification of DD_TRACE_METHODS.
 
-    DD_TRACE_METHODS is specified to be FullyQualifiedClassOrModuleName[comma-separated-methods]
+def _parse_trace_methods(raw_dd_trace_methods: str) -> List[Tuple[str, str]]:
+    """Return a list of the module,methodname tuples to trace based on the
+    specification of DD_TRACE_METHODS.
+
+    DD_TRACE_METHODS is specified to be FullyQualifiedModuleName:comma-separated-methods;...
+    The square bracket notation will be deprecated in favor of this new syntax
```

Comment:
This line doesn't make sense in the context of the rest of this file. It's a true statement, but `_parse_trace_methods` doesn't know anything about "square bracket notation" - that's `_parse_legacy_trace_methods`' concern.